docs/source/apis/saas_api.rst

```diff
@@ -0,0 +1,137 @@
+SaaS API Deployment Guide
+=========================
+
+This documentation serves as a guide for Unstructured SaaS API users to register and manage API Key, as well as the billing information. It contains all the necessary instructions and resources to facilitate a smooth setup and deployment process of Hosted API on Unstructured infrastructure.
+
+
+New User Sign-up
+----------------
+
+1. Navigate to the `Unstructured SaaS API <https://unstructured.io/api-key-hosted>`_ page on our website.
+2. Fill out the registration form with your details:
+
+   - **First Name**: Enter your given name.
+   - **Last Name**: Enter your family or surname.
+   - **Email**: Provide a valid email address where we can send you confirmations and communications.
+   - **Company**: Enter the name of your company or organization.
+
+3. Optional Preferences:
+
+   - [] Check the box to sign up for future marketing emails and product updates.
```

Comment:
These look weird in the PDF version because you can't click them and they don't render as boxes.